content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -135,14 +148,32 @@ Some default values for mesh shape dimensions have also been changed to be more
 [`2d_shapes`]: https://bevyengine.org/examples/2D%20Rendering/2d-shapes/
 [`3d_shapes`]: https://bevyengine.org/examples/3D%20Rendering/3d-shapes/
 
-### Gizmos
+#### Gizmos
 
 <div class="release-feature-authors">authors: @RobWalt</div>
 
-TODO
+All of the primitives mentioned above can also be rendered with `Gizmos`. For
+this we designed a generic API which should feel very familiar if you have used
+`Gizmos` before. There are two new methods now:
+
+* [`gizmos.primitive_2d(primitive, position, angle, color)`]
+* [`gizmos.primitive_3d(primitive, position, rotation, color)`]
+
+Based on the primitive there might be some additional configuration options
+similar to existing `Gizmos` drawing methods, e.g. here is a small showcase of
+the [`SphereBuilder`], which offers a `segments` method to control the level of
+detail of the sphere.
```

Comment:
- Imo we should avoid abbreviations like "e.g." in texts like this.
- It's unclear where `SphereBuilder` comes from since it's not visible in code.

A more direct and hopefully clearer version:

```suggestion
Some primitives can have additional configuration options similar to existing [`Gizmos`] drawing methods.
For example, calling [`primitive_3d`] with a [`Sphere`] returns a [`SphereBuilder`], which offers a `segments` method
to control the level of detail of the sphere.
```